File: Week 6/Problem Set 6/sentimental-mario-less/mario.py
def main():
    height = get_height()
    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")
        print("",end ="\n")

def get_height():
    while True:
        try:
            n = int(input("Enter height: "))
            if n > 0 and n < 9:
                return n

        except ValueError:
            print("Not an Integer")


main()

File: Week 1/Problem Set 1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height, row;

    //declare height limit for program.
    printf("Enter Height of Triangle (1-8). \n");

    //prompt until user inputs integer from 1-8.
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    printf("\n");

    //set row
    for (row = 0; row < height; row++)
    {

        //print space with following computation
        for (int i = 0; i < height  - row - 1; i++)
        {
            printf(" ");
        }

        //print hashe with following computation
        for (int j = 0; j <= row; j++)
        {
            printf("#");
        }

        //print new line when one row completes
        printf("\n");
    }
    return 0;
}